backend/homework/3/src/main/java/org/example/hospitalmanagement/Patient.java

```diff
@@ -0,0 +1,47 @@
+package org.example.hospitalmanagement;
+
+import org.example.billing.Billing;
+import org.example.billing.HealthInsurancePlan;
+import org.example.billing.PlatinumPlan;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Patient extends User {
+    private static final Logger logger = LoggerFactory.getLogger(Patient.class);
+
+    private long patientId;
+    private boolean insured;
+    private HealthInsurancePlan insurancePlan;
+
+    public void setInsured(boolean insured) {
+        this.insured = insured;
+    }
+
+    public HealthInsurancePlan getInsurancePlan() {
+        return insurancePlan;
+    }
+
+    public void setInsurancePlan(HealthInsurancePlan insurancePlan) {
+        this.insurancePlan = insurancePlan;
+    }
+
+    public static void main(String[] args) {
```

Comment:
Write the main method in a separate driver class. Let the Patient class have single responsibility.